neurobooth_os/gui.py

```diff
@@ -57,11 +54,10 @@ def emit(self, record):
         buffer = f'{BUFFER}\n{str(record)}'.strip()
         window['log'].update(value=buffer)
 
-BUFFER = ''
-
-logger = setup_log(__name__, sg_handler = Handler().setLevel(logging.DEBUG))
 
+BUFFER = ''
```

Comment:
What is this buffer? Just the message prefix? A comment may help here. I may also be missing what is happening with it elsewhere in the code.

Reply:
Not sure. It was there when i got there. Since PyCharm didn't flag it as unused I left it alone. 

Looking again, it seems to be only used in emitting logs to the GUI window: 

    def emit(self, record):
        global BUFFER
        buffer = f'{BUFFER}\n{str(record)}'.strip()
        window['log'].update(value=buffer)

I don't see why it should be global so I'll move it into the method.